contrib/docs/tutorials/aws-alb-aad-oidc-auth.md

```diff
@@ -182,9 +182,8 @@ Use the following `auth` configuration when running Backstage on AWS:
 auth:
   providers:
     awsalb:
-      development:
```

Comment:
awsalb provider doesn't use the authEnv:

https://github.com/getndazn/backstage/blob/00b5d57ab3bc139842f307cf4526dc39687afb81/plugins/auth-backend/config.d.ts#L112-L115